compiler/cpp/src/interpreter/interpreter.cpp

```diff
@@ -92,11 +92,18 @@ bool evaluateCompStatement(CompStmtExprAST* ast, const json& data) {
     json keyValue = getJsonValueFromKey(ast->GetKey(), data);
     if (keyValue.is_string()) {
         auto value = keyValue.get<std::string>();
-        if (auto* expectedValue = dynamic_cast<StringExprAST*>(ast->GetValue().get())) {
+        if (auto *expectedValue = dynamic_cast<StringExprAST*>(ast->GetValue().get())) {
```

Comment:
I prefer it like this too.

Reply:
Argue with @keksklauer4 about that. He told me, that he likes the other way better. A matter of taste ... ;)